src/ms.tex

```diff
@@ -184,15 +197,18 @@ \subsection{Highlighted Feature Development} \label{sec:core-features}
 
 \section{Major Updates in the Astropy Project} \label{sec:project-updates}
 
-
 \subsection{Project governance} \label{sec:project-governance}
 
-New CoCo and election overview.
+\secauthor{Erik Tollerud}
```

Comment:
@eteq : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

Reply:
Happy to write this!